### PR TITLE
All changes in one PR

### PR DESCRIPTION
**Pull request by thomasjpfan**

### CODE REVIEW

File: slides/aml-06-linear-models-classification/index.html [hunk 1]
```diff
@@ -690,8 +690,8 @@
 # In sckit-learn
 
 - OvO: only SVC
-- OvR: default for all linear models, even LogisticRegression (deprecated)
-- `LogisticRegression(multinomial=True)`
+- OvR: default for all linear models
```
Comment (amueller) [inline slides/aml-06-linear-models-classification/index.html]: shouldn't it now say "except for logistic regression"? calling the binary one OvR is a weirdness of our implementation and should be hidden from users.

File: slides/aml-06-linear-models-classification/index.html [hunk 2]
```diff
@@ -1347,19 +1347,22 @@
 
 .smaller[
 ```
-gamma : float, optional (default = "auto")
+gamma{‘scale’, ‘auto’} or float, optional (default=’scale’)
```
Comment (amueller) [inline slides/aml-06-linear-models-classification/index.html]: shouldn't there be a colon?

Comment (thomasjpfan) [reply slides/aml-06-linear-models-classification/index.html]: Interesting how copy and paste directly from the HTMl does not capture the `:`.

**Comment by amueller** [comment]
Thanks!

**Pull request merged**